### DISCUSSION TITLE
How can we access User Accounts Setup?

### DISCUSSION THREAD

**Discussion by emaalouf** [notrinos/NotrinosERP#3] [General]
Hi, thank you for this amazing job! I am looking to get   User Accounts Setup
How to access admin in the demo!

**Comment by notrinos** [comment]
Hello @emaalouf 
Admin access role in the demo is not provided.
If you want to play with admin or any access roles you should install the system in your machine then practice in Setup -> User Account Setup and Setup -> Access Setup
Thank you
